Makefile

```
@@ -1,16 +1,26 @@
 
 CC=gcc
-CFLAGS=-Og -g -march=native -std=c11 -Wall -Wno-stringop-truncation -I.
+# If we build with ddc
+CFLAGS=-Og -g -march=native -std=c11 -Wall -Wno-stringop-truncation -I. -lddcutil
```

```suggestion
CFLAGS=-Og -g -march=native -std=c11 -Wall -Wno-stringop-truncation -I. -lddcutil -lm
```
We need math support atm